src/main/scala/RigPlugin.scala

```diff
@@ -133,7 +133,7 @@ object common {
           "-Xlint:package-object-classes",
           "-Xlint:unsound-match",
           "-Xlint:stars-align"
-        )
+        ) ++ (if(n >= 12) {Seq("-Ypartial-unification")} else Seq())
```

Comment:
Could we make this the same case match style as earlier lines? Would like to be consistent.